client/my-sites/upgrades/index.js

```diff
@@ -252,6 +252,11 @@ module.exports = function() {
 		);
 
 		page(
+			'/checkout/no-site',
```

Comment:
Also maybe we should come up with a better path fragment? https://wordpress.com/checkout/no-site doesn't look really nice for users.